ci/docker/README.md

```diff
@@ -6,24 +6,3 @@ These are _not_ the Dockerfiles that build the `rayproject/ray` releases. These
 are found in the `/docker` directory under the root.
 
 The Dockerfiles are hierarchical and will be built in different places during a CI run.
-
```

Comment:
perhaps delete this file too since the remaining texts now seem in-complete and this readme is something we don't want to maintain (no longer mentioning how the ci images are built)

Reply:
done.